test/Target/ExportSMTLIB/attributes.mlir

```diff
@@ -50,6 +50,49 @@ smt.solver () : () -> () {
   smt.assert %3
 
 
+  // CHECK: (assert (let (([[V21:.+]] (exists (([[V22:.+]] Int) ([[V23:.+]] Int))
+  // CHECK:                           ( ! (let (([[V24:.+]] (= [[V22]] [[V23]])))
+  // CHECK:                           (let (([[V25:.+]] (=> [[V24:.+]] true)))
+  // CHECK:                           [[V25:.+]])) 
+  // CHECK                            :pattern ( ( let (([[V26:.+]] (= [[V22:.+]] [[V23:.+]])))
+  // CHECK:                   [[V26:.+]]))))))
```

Comment:
> `[[V25:.+]]` matches and stores a string, `[[V25]]` checks if the last string stored to `V25` is present here. The use of the earlier should be replaced with the latter to make this test stronger.

sorry this is not entirely clear to me and I haven't fixed it yet, should I use `[[V25]]` instead of `[[V25:.+]]`?

Reply:
nvm got it! fixing it now :)

Reply:
> > `[[V25:.+]]` matches and stores a string, `[[V25]]` checks if the last string stored to `V25` is present here. The use of the earlier should be replaced with the latter to make this test stronger.
> 
> sorry this is not entirely clear to me and I haven't fixed it yet, should I use `[[V25]]` instead of `[[V25:.+]]`?

I got the issue with `[[V25]]` and is fixed now, but then I believe that `[[V26:.+]]` at line 58 should also become `[[V26]`. However, when I replace this at line 58 the tests stop working because `[[V26]] is undefined` and I really can not understand why, do you have any idea? Thanks a lot :)